.github/workflows/simorgh-amp-validator.yml

```
@@ -0,0 +1,34 @@
+name: Simorgh CI - AMP Validation
```

Could we add bbc-a11y to this too?

```suggestion
name: Simorgh CI - AMP Validation & bbc-a11y
```